### PR TITLE
[Knowledge][Test] Add taxonomy for instruct-lab 

### PR DESCRIPTION
**Pull request by harshit-sh**
**Describe the contribution to the taxonomy**
- Train the model with instruct-lab related 'knowledge'.

**Contribution checklist**

- [x] The contribution was tested with `ilab generate`
- [x] No errors or warnings were produced by `ilab generate`
- [x] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [x] The `qna.yaml` file contains at least 5 `seed_examples`
- [x] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [x] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [x] Content does not include PII or otherwise sensitive or confidential information
- [x] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

**Review by bjhargrave — CHANGES_REQUESTED** [review]
Please fix lint issues.

File: knowledge/technical_manual/instructlab/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+created_by: Harshit
```
Comment (jjasghar) [inline knowledge/technical_manual/instructlab/qna.yaml:1]: please put your GitHub Username here.

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please fix the created_by, then we can run the linter.

**Comment by bjhargrave** [comment]
@mergifyio rebase

**Comment by bjhargrave** [comment]
Please add

```yaml
version: 2
```

to the start of the qna.yaml file. This is now a requirement of the schema validation process during linting.

File: knowledge/technical_manual/instructlab/qna.yaml (hunk 1 above)
Comment (harshit-sh) [reply knowledge/technical_manual/instructlab/qna.yaml]: Updated the line.

**Pull request closed without merging**